game/theater/theatergroundobject.py

```diff
@@ -481,18 +488,64 @@ def __init__(
             dcs_identifier="AA",
             sea_object=False,
         )
-        # Set by the SAM unit generator if the generated group is compatible
-        # with Skynet.
-        self.skynet_capable = False
+        self.aa_ranges: List[AirDefenseRange] = [aa_range]
 
     @property
+    # Returns the TGO Name
     def group_name(self) -> str:
-        if self.skynet_capable:
-            # Prefix the group names of SAM sites with the side color so Skynet
-            # can find them.
-            return f"{self.faction_color}|SAM|{self.group_id}"
+        return super().group_name
+
+    # Returns the name of one single group
+    def groups_name(self, group: Group) -> str:
```

Comment:
Only called from within an iteration over the groups. Make this take an int index instead and enumerate the loop at the callsite rather than searching the list for the group?